packages/twenty-front/src/modules/ui/layout/page/components/PageBody.tsx

```diff
@@ -22,6 +22,7 @@ const StyledMainContainer = styled.div`
   padding-right: ${({ theme }) => theme.spacing(3)};
   padding-left: 0;
   width: 100%;
+  height: 100%;
```

Comment:
Let's not modify PageBody to fix a problem for settings because it is used everywhere and the side effects will be hard to predict

Reply:
Especially since we spent a lot of time adjusting the global app layout and this particular CSS configuration might be the result of it.